template/src/lib/ui/stories/common/Palette.stories.js

```diff
@@ -50,6 +50,24 @@ const ColorTemplate = ({ colorVariant, shade, ...args }) => {
           {colorDef.dark}
         </Typography>
       </Box>
+      {colorDef.relatedDark && (
+        <Box style={{ backgroundColor: colorDef.relatedDark, ...boxStyle }} {...args}>
```

Comment:
What color is relatedDark or relatedLight?

Reply:
New palette colors: for primary, secondary, etc https://app.clubhouse.io/cartoteam/story/119457/qa-storybook-palette

Reply:
Ok. But maybe we should keep only colors from the MaterialUI theme, so the devs will know what to expect. If not, we should add them to the documentation and make it clear for them.

Reply:
@alasarr @javigil-geo I think that we can go with palette for now and discuss it

Reply:
Agree with @neokore, but I'm going to merge and @Josmorsot will create a PR later removing the colors